src/webapp/azext_webapp/__init__.py

```diff
@@ -24,24 +17,12 @@ def __init__(self, cli_ctx=None):
 
```

Comment:
You will need to remove the `min_profile` annotation here for CLI core version 2.0.67 and up or your extension will break (that property is being removed). You will need an index to that last working extension for 2.0.46-2.0.66. Please talk to @limingu.

Reply:
This is still not done.